index.js

```diff
@@ -330,7 +340,32 @@ function redisStore(args) {
       if (err) {
         return cb && cb(err);
       }
-      conn.keys(pattern, handleResponse(conn, cb));
+
+      // Use an object to dedupe as scan can return duplicates
+      var keysObj = {};
+      var count = 100;
```

Comment:
I hard-coded it, because there are no options in the `del()` signature, right now. Adding options might be a breaking change. Should it be a global option?